### DISCUSSION TITLE
A Python Brasil 2022 precisa de ajuda!

### DISCUSSION THREAD

**Discussion by julianyraiol** [pythonbrasil/pybr2022-org#221] [General]
A  [Python Brasil](https://github.com/pythonbrasil/pybr2022-org) desse ano terá sua primeira edição no Norte do país, em Manaus! O evento acontecerá nos dias 17-23 de outubro.

Infelizmente, estamos com poucos patrocinadores até o momento e ainda estamos longe da meta para as despesas mínimas do evento.

Você trabalha em alguma empresa que costuma patrocinar eventos? Ou conhece alguém que trabalha em uma? Comenta aqui nessa discussão, envie o plano de patrocínio abaixo ou entre em contato com a gente (telegram:  @julianyraiol, @wasp_lahis, @jaja_jp, ou aqui na discussão mesmo)

- Plano em português: [link](https://2022.pythonbrasil.org.br/static/patrocinio_python_brasil_ptbr-6ed512a41579cfea596d03d6453816dd.pdf)
- Plano em inglês: [link](https://2022.pythonbrasil.org.br/static/patrocinio_python_brasil_en-3cee1cf87e43dc340ec8c880ee19bc1a.pdf)

Obrigada, gente :heart:

**Comment by wasp-lahis** [comment]
@pythonbrasil/pybr2012-org @pythonbrasil/pybr2014-org @pythonbrasil/pybr2015-org @pythonbrasil/pybr2016-org @pythonbrasil/pybr2017-org @pythonbrasil/pybr2018-org @pythonbrasil/pybr2019-org @pythonbrasil/pybr2020-org
